bower.json

```diff
@@ -10,5 +18,8 @@
         "node_modules",
         "resources",
         "tasks"
+    ],
+    "dependencies": [
+        "jquery": ">=2.0.0"
```

Comment:
@shehi bootstrap-progressbar will work for most settings without bootstrap (depends on your styles)